priv/repo/migrations/20231229010413_add_tenant_allow_list.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Supavisor.Repo.Migrations.AddTenantAllowList do
+  use Ecto.Migration
+
+  def change do
+    alter table("tenants", prefix: "_supavisor") do
+      add(:allow_list, {:array, :string}, null: false, default: ["0.0.0.0/0", "::/0"])
```

Comment:
It's fine the way it is, but just for completeness, PG has these [Network Address Types](https://www.postgresql.org/docs/current/datatype-net-types.html) that may be helpful to keep the data integrity at the DB level :)

Reply:
Ah yeah nice! Will make another pass with this in a bit.